src/transform/benches/README.md

```diff
@@ -0,0 +1,83 @@
+# `mzbench-opt`
+
+The `mzbench-opt` package contains optimizer benchmarking scripts.
+
+## Installation (`pyenv`)
+
+Run the following commands only once on each host.
+
+```bash
+pyenv virtualenv 3.9.5 mzbench-opt
+```
+
+Run the following commands whenever the contents of this folder change.
+
+```bash
+# activate mzbench-opt environment
+pyenv virtualenv-delete mzbench-opt
+pyenv virtualenv 3.9.5 mzbench-opt
+
+# install package
+pyenv activate mzbench-opt
+pip install -e .
+
+# install zsh shell completion
+mzbench-opt --install-completion zsh
+rm -f ~/.zcompdump; compinit
+```
+
+## Run Benchmarks
+
+To run a benchmark you need to:
+
+1. Intialize the database schema for your test scenario.
+1. Run the workload of your test scenario.
+
+These two steps are modeled by the `init` and `run` commands.
+For example, for the `tpch` scenario, type
+
+```bash
+mzbench-opt init tpch
+mzbench-opt run tpch --repository="$HOME/mzbench-opt"
+```
+
+The results of benchmark `run` are stored in the OS temp folder by default, but here we set the location to `$HOME/mzbench-opt` using the `--repository` option.
```

Comment:
nit: could also make the current working directory the default (which might make it more explicit that files are being written out)